docs/writing-docs/user-tutorials.md

```diff
@@ -74,6 +74,16 @@ In context tutorials, sometime referred to as "recipes", are tutorials that are
 
 Click on the ``lab`` icon in the **Explorer** view to open any markdown file (``.md``) as a tutorial in a new tab.
 
+### ~ alert
+
+#### Tutorial caching
+
+MakeCode uses a local caching policy for tutorials to reduce interaction with website services. On first use, tutorial content is retrieved from a MakeCode website and then reused from the local cache when a tutorial is run another time. A requested tutorial will refresh from the website when its cache retention period expires.
+
+This caching policy can present a problem if you're developing a tutorial and want to review the recent changes. When you run the tutorial to check your changes, they might not appear and you only see content you viewed the first time. In order to see and test new changes you've published for your tutorial, it's recommended that your view them in a **new anonyomous / incognito** browser window.
```

Comment:
not quite sure where to put this, but if they are writing Github tutorials specifically, they will need to create a release using the MakeCode Github integration to force-clear the cloud cache (documentation here: https://arcade.makecode.com/github/release)

Reply:
See if my in note here https://github.com/microsoft/pxt/pull/7481/commits/f150b055f379e645dbf9ff5d468e586cadb37717 is legit.

Reply:
looks awesome, thanks galen!!